idl/group0.idl.hh

```diff
@@ -20,6 +20,7 @@ struct group0_info {
     raft::group_id group0_id;
     raft::server_id id;
     gms::inet_address ip;
+    std::vector<service::discovery_peer> peers;
```

Comment:
Re the last commit message:

why doesn't A (eventually) learn about B's IP address through gossip? (in both cases)

Reply:
Exactly. May be this is a leftover before moving gossiper start before group0 joining?

Reply:
Adding this just improves test time. I wrote in slack today, from 2:20 down to 1:20-1:40